LearnositySDKUnitTests/InitTests.cs

```diff
@@ -12,6 +11,10 @@ namespace LearnositySDKUnitTests
     [TestClass]
     public class InitTests
     {
+        private const string ConsumerKey = "yis0TYCu7U9V4o7M";
+        private const string ConsumerSecret = "74c5fd430cf1242a527f6223aebd42d30464be22";
+        private const string SecurityString = "{\"consumer_key\":\"" + ConsumerKey + "\",\"domain\":\"localhost\",\"timestamp\":\"20140626-0528\"}";
+
```

Comment:
Since this information is already available in the `TestRequest` class, perhaps you can make getters from there instead of duplicating this information across multiple files.

Reply:
This has been fixed.